llvm/include/llvm/Support/Endian.h

```diff
@@ -24,7 +24,11 @@
 namespace llvm {
 namespace support {
 
-enum endianness {big, little, native};
+enum endianness {
```

Comment:
Would be nice to make it `enum class` at some point

Reply:
Yes, indeed.  I'm planning to do that.  Thank you for the review!